### ISSUE TITLE
default_webserver_config.py isn't compatible with the latest changes

### ISSUE THREAD

**Issue by zachliu** [apache/airflow#36702]
### Apache Airflow version

2.8.0

### What happened?

[default_webserver_config.py
](https://github.com/apache/airflow/blob/2.8.0/airflow/config_templates/default_webserver_config.py) is not compatible with 2.8.X versions due to https://github.com/apache/airflow/pull/35572

specifically, [these imports](https://github.com/apache/airflow/blob/db2b75c233e3e3c59ec9d0563b93ddbe733ad0bf/airflow/config_templates/default_webserver_config.py#L25-L28) are not valid anymore

```python
# from airflow.www.fab_security.manager import AUTH_LDAP
# from airflow.www.fab_security.manager import AUTH_OAUTH
# from airflow.www.fab_security.manager import AUTH_OID
# from airflow.www.fab_security.manager import AUTH_REMOTE_USER
```

they should be

```python
# from airflow.auth.managers.fab.security_manager.override import AUTH_LDAP
# from airflow.auth.managers.fab.security_manager.override import AUTH_OAUTH
# from airflow.auth.managers.fab.security_manager.override import AUTH_OID
# from airflow.auth.managers.fab.security_manager.override import AUTH_REMOTE_USER
```

### How to reproduce

for oauth users just keep using the old `webserver_config.py`
```
from airflow.www.fab_security.manager import AUTH_OAUTH
```
will cause
```
ImportError: cannot import name 'AUTH_OAUTH' from 'airflow.www.fab_security.manager' (/home/zach/KeplerGroup/KIP-Airflow/.venv/lib/python3.11/site-packages/airflow/www/fab_security/manager.py)
```

### Operating System

Debian GNU/Linux 10 (buster)

### Deployment

Other Docker-based deployment

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
cc: @vincbeck

**Comment by vincbeck** [comment]
It should have been fixed in #35302

**Comment by vincbeck** [comment]
Oh, some are missing, let me add them

**Comment by vincbeck** [comment]
See fix: #36719

**Comment by vincbeck** [comment]
You might want to add the fix to 2.8.1
